plugin-ci/orb.yml

```diff
@@ -10,7 +11,7 @@ orbs:
 executors:
   default:
     docker:
-      - image: circleci/golang:1.16.0-node
+      - image: cimg/go:1.19-node
```

Comment:
Do we even need the `node` version if `npm` is installed via `nvm`?

Reply:
@hanzei I was thinking the same, but this comment made me think otherwise https://discuss.circleci.com/t/circleci-2-0-installing-nvm-on-docker-executor-fails/38011/2. I can try it without `-node` if you'd like

Reply:
@hanzei seems to work correctly

- https://github.com/mattermost/mattermost-plugin-zoom/commit/71df454e129907a92f42c06fd4f071b5f696b7eb
- https://app.circleci.com/pipelines/github/mattermost/mattermost-plugin-zoom/2056/workflows/f904bb8c-e449-49cd-af7b-99bd94ae8694/jobs/6418

Reply:
Neat, let's use the bare minimum.